src/views/SpikeAmplitudes/SpikeAmplitudesView.tsx

```diff
@@ -150,29 +150,29 @@ const SpikeAmplitudesViewChild: FunctionComponent<ChildProps> = ({data, timeseri
         return {yMin, yMax}
     }, [series])
 
-    const pixelTransform = use2dPanelDataToPixelMatrix(
-        pixelsPerSecond,
-        visibleTimeStartSeconds,
-        amplitudeRange.yMin,
-        amplitudeRange.yMax,
-        ampScaleFactor,
-        panelHeight,
-        true
-    )
-
+    const pixelTransform = use2dScalingMatrix({
+        totalPixelWidth: panelWidth,
+        totalPixelHeight: panelHeight,
+        dataXMin: visibleTimeStartSeconds,
+        dataXMax: visibleTimeEndSeconds,
+        dataYMin: amplitudeRange.yMin,
+        dataYMax: amplitudeRange.yMax,
+        yScaleFactor: ampScaleFactor,
+    })
+
+    // TODO: Figure out how to encapsulate y-axis management into timescrollview
     const yTicks = useYAxisTicks({ datamin: amplitudeRange.yMin, datamax: amplitudeRange.yMax, userSpecifiedZoom: ampScaleFactor, pixelHeight: panelHeight })
     const yTickSet = useProjectedYAxisTicks(yTicks, pixelTransform)
 
     const panels: TimeScrollViewPanel<PanelProps>[] = useMemo(() => {
+        const pixelZero = getYAxisPixelZero(pixelTransform)
```

Comment:
This is going to be ugly no matter what we do (see the prior version l. 172) but at least we can encapsulate it.